packages/integrations/preact/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import preact, { type PreactPluginOptions as VitePreactPluginOptions } from '@preact/preset-vite';
+import { preact, type PreactPluginOptions as VitePreactPluginOptions } from '@preact/preset-vite';
```

Comment:
Removed the default export here, the types were annoying. Importing it like this works, the tests pass and the types are much easier